src/components/strings/helpers.js

```diff
@@ -0,0 +1,14 @@
+import { uploadDesktop, uploadMobile } from './'
+import { isDesktop } from '../utils'
+
+export const uploadCopy = (method, documentType, side) => {
```

Comment:
what does this do?

Reply:
It parses the the strings file to get you the strings used for a given screen. This now depends on mobile vs desktop, document type, document side, document vs selfie.

Reply:
feels a bit weird to see it outside of a component since it's presentation logic. but then again, apart from that I don't have strong arguments against it.